typespec-extension/src/code-model-builder.ts

```diff
@@ -1018,7 +1020,7 @@ export class CodeModelBuilder {
     } else {
       // schema
       let schema;
-      const sdkType = getClientType(this.sdkContext, param.param);
+      const sdkType = getNonNullSdkType(getClientType(this.sdkContext, param.param));
```

Comment:
For parameter, the `nullable` is from `param.param.type` (the raw type).

I didn't change the logic.